### DISCUSSION TITLE
Open File Above/Below shortcut

### DISCUSSION THREAD

**Discussion by anaclumos** [notaapp/talk#315] [Feature Requests] [answered]
It would be nice to have a shortcut to open the file above/below shown on the sidebar!

For example, if the files are sorted like the following on the left pane,

```
note1
note2
note3
```

If I am currently at `note2` and hit the `open note above` shortcut, it will bring me to `note1`, and if I press the `open note below` shortcut, it will get me to `note3`.

**Comment by anaclumos** [comment]
**Navigating back to the previously opened page** will also feel nice. For example, `command+[` and `command+]` will act like backward/forward buttons on the browser. This is different from opening the next tab with `command+shift+[`, because users may not open the page chronologically on the top tab bars. This is also a universal behavior to many documentation tools like Obsidian and Docusaurus.

**Comment by astoilkov** [comment, marked as answer]
We will be implementing `command+[` and `command+]` shortcuts in the next version.

For `open note above/below` we still don't have plans but we will discuss your suggestion.

Thanks!
